### PR TITLE
feat(chips): add alarm control panel chip

### LINKED ISSUE #105: Alarm chip icon color not honored
**Describe the bug**

Setting the icon color does nothing for the alarm chip icon

**To Reproduce**

1. Add an alarm chip
2. Set the icon color to any color from the icon color dropdown
3. Click save

**Expected behavior**

Expect to see the color set on the icon

**Screenshots**

image

### PR DESCRIPTION
**Pull request by bemble**
chip-alarm-control-panel-lightchip-alarm-control-panel-dark
Add alarm control panel that displays icon and color like the card.

Also added chip name translation, if PR is validated, we have to update translations in #130 

fix #105

### CODE REVIEW

File: src/utils/lovelace/element-editor.ts [hunk 1]
```diff
@@ -284,6 +284,7 @@ export abstract class MushroomElementEditor<T> extends LitElement {
                 configElement = await this.getConfigElement();
 
                 if (configElement) {
+                    console.log(configElement);
```
[the remark this answers is not in the document]
Comment (bemble) [reply src/utils/lovelace/element-editor.ts:287]: Oh noooooooo, wait for the next PR ^^!

File: src/cards/chips-card/chips/alarm-control-panel-chip.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    }
+
+    static get styles(): CSSResultGroup {
+        return css`
+            mushroom-chip {
+                cursor: pointer;
+            }
+            ha-icon.active {
+                color: var(--color);
+            }
```
[the remark this answers is not in the document]
Comment (bemble) [reply src/cards/chips-card/chips/alarm-control-panel-chip.ts:113]: You're right, I'll solve it.

**Comment by piitaya** [comment]
For the translation, if the key is not present in the file, it will fallback to english so it's not a problem 🙂

File: .hass_dev/configuration.yaml [hunk 3]
```diff
@@ -92,7 +92,7 @@ alarm_control_panel:
               - alarm_control_panel.alarm_panel_1
               - alarm_control_panel.alarm_panel_2
         disarm:
-          service: alarm_control_panel.alarm_arm_home
+          service: alarm_control_panel.alarm_disarm
```
Comment (piitaya) [inline .hass_dev/configuration.yaml:95]: 👍

File: src/cards/chips-card/chips-card-chips-editor.ts [hunk 4]
```diff
@@ -105,7 +105,7 @@ export class ChipsCardEditorChips extends LitElement {
                     ${CHIP_LIST.map(
                         (chip) => html`
                             <paper-item data-type="${chip}" }
```
[the remark this answers is not in the document]
Comment (bemble) [reply src/cards/chips-card/chips-card-chips-editor.ts:107]: Thanks for pointing that, I'll create a tech issue.

**Pull request merged**